src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1981,4 +2001,27 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public boolean hasWillRaidTonight()
+    {
+        return willRaidTonight;
+    }
+
+    public void setWillRaidTonight(final Boolean willRaid)
+    {
+        willRaidTonight = willRaid;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2696)